lms/djangoapps/teams/static/teams/js/teams_features.js

```diff
@@ -61,19 +61,43 @@
     };
 
     function buttonAddMembers(staff, url){
-        var button = $("<button class='action action-primary'>Add Members</button>");
+        var button = $("<button class='btn btn-secondary'>Add Members</button>");
         var input = $("<input type='file' name='fileUpload' style='display: none;'accept='text/csv'/>")
         if(!$(".page-header-secondary").children()[0] && staff){
             $(".page-header-secondary").append(input);
             $(".page-header-secondary").append(button);
 
             input.fileupload({
                 url: url,
-                done:function(){
+                done:function(e, data){
                     $(".page-header-secondary").empty();
                     buttonAddMembers(staff, url);
+                    var errors = data.result.errors;
+                    var success = data.result.success;
+                    var container = $("<div title='Result'></div>");
+                    var list = $("<ul></ul>");
+
+                    for (var key in errors){
+                        var item = $("<li style='color: red; font-size:12px;'>Error in "+key+" = "+errors[key]+"</li>");
```

Comment:
what do you think about using inline css styles @andrey-canon @diegomillan ? Can Adonis give us a hand here?

Reply:
check, it was removed